### PR TITLE
VPA admission controller e2e test.

### PR DESCRIPTION
**Pull request by schylek**

### CODE REVIEW

File: vertical-pod-autoscaler/e2e/admission_controller.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		err = framework.WaitForDeploymentComplete(c, d)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		podList, err := framework.GetPodsForDeployment(c, d)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		// Originally Pods had 100m CPU, but admission controller should change it to recommended 200m CPU
+		for _, pod := range podList.Items {
+			gomega.Ω(pod.Spec.Containers[0].Resources.Requests[apiv1.ResourceCPU]).Should(gomega.Equal(parseQuantityOrDie("200m")))
```
Comment (bskiba) [inline vertical-pod-autoscaler/e2e/admission_controller.go]: I think Kubernetes uses the Expect(sth).To(Equal(sth else) idiom (https://onsi.github.io/gomega/#making-assertions)

File: vertical-pod-autoscaler/e2e/admission_controller.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+		d := hamsterDeployment(f, cpuQuantity, memoryQuantity)
+		d, err = c.ExtensionsV1beta1().Deployments(ns).Create(d)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		err = framework.WaitForDeploymentComplete(c, d)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		podList, err := framework.GetPodsForDeployment(c, d)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		// Originally Pods had 100m CPU, but admission controller should change it to recommended 200m CPU
```
Comment (bskiba) [inline vertical-pod-autoscaler/e2e/admission_controller.go]: Should we check what happened to memory?

File: vertical-pod-autoscaler/e2e/admission_controller.go (hunk 1 above)
Comment (schylek) [reply vertical-pod-autoscaler/e2e/admission_controller.go]: Done.

File: vertical-pod-autoscaler/e2e/admission_controller.go (hunk 2 above)
Comment (schylek) [reply vertical-pod-autoscaler/e2e/admission_controller.go]: Added assertions for memory check.

**Pull request merged**